### PR TITLE
Exclude slf4j-jboss-logmanager from quarkus-vertx for logging

### LINKED ISSUE #2933: Data plane logging is not working
**Describe the bug**

See dispatcher logs here https://storage.googleapis.com/knative-prow/pr-logs/pull/knative-sandbox_eventing-kafka-broker/2930/reconciler-tests_eventing-kafka-broker_main/1620015267549548544/artifacts/knative-eventing/kafka-broker-dispatcher.

There are no logs for our classes and the logs are not serialized in the usual structured form

 **Expected behavior**

Structured logs from our classes

 **To Reproduce**

Run data plane

 **Knative release version**

`main`

 **Additional context**
Add any other context about the problem here such as proposed priority

### PR DESCRIPTION
**Pull request by creydr**
Fixes #2933

### CODE REVIEW

**Comment by pierDipi** [comment]
Nice, logging is back! 

https://storage.googleapis.com/knative-prow/pr-logs/pull/knative-sandbox_eventing-kafka-broker/2934/channel-reconciler-tests-sasl-plain_eventing-kafka-broker_main/1620773661101264896/artifacts/knative-eventing/kafka-broker-dispatcher

you need to run `hack/update-codegen.sh` or (more quickly) `./mvnw package -DskipTests`

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

File: data-plane/core/pom.xml [hunk 1]
```diff
@@ -96,6 +96,10 @@
           <groupId>org.osgi</groupId>
           <artifactId>org.osgi.core</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>org.jboss.slf4j</groupId>
```
Comment (matzew) [inline data-plane/core/pom.xml:100]: yummy

**Pull request merged**